lib/inferno/dsl/fhir_resource_validation.rb

```diff
@@ -73,19 +73,23 @@ def igs(*validator_igs)
         # there is no check that the fields are correct.
         #
         # @example
+        #   # Passing fields in a block
         #   fhir_resource_validator do
         #     url 'http://example.com/validator'
         #     cli_context do
         #       noExtensibleBindingMessages true
+        # .     allowExampleUrls true
```

Comment:
```suggestion
        #       allowExampleUrls true
```
MacOS auto-inserting a . when you type two spaces strikes again